synapse/cortex.py

```diff
@@ -935,6 +934,11 @@ class Cortex(s_cell.Cell):  # type: ignore
             'description': 'Set the default lmdb:map_async value in LMDB layers.',
             'type': 'boolean'
         },
+        'layer:lmdb:max_replay_log': {
+            'default': True,
+            'description': 'Set the max size of the replay log for all layers.',
+            'type': 'boolean'
```

Comment:
Should this be an int rather than a bool to make it actually tunable?

Reply:
Yes